app/src/main/kotlin/cz/covid19cz/app/bt/BluetoothRepository.kt

```diff
@@ -276,13 +279,17 @@ class BluetoothRepository(
     private fun onScanIosOnBackgroundResult(result: ScanResult) {
         L.d("Scan All onResult")
         if (result.scanRecord?.bytes != null) {
```

Comment:
tady by pomohlo kdyz by misto toho ifu bylo `device.scanRecord?.let {}. pak by to uvnitr bylo nonnull a mohli bysme dat do haje ty dva vykricniky 😃

Reply:
Fixed.